File: Data Structure/Array Or Vector/Linear Search/SolutionByMashiyat.c
#include <stdio.h>
 void main()
 {
     int array[100],search,c,n;
     printf("Enter number of elements in array\n");
     scanf("%d",&n);
     printf("Enter %d integers\n",n);
     for(c=0;c<n;c++)
        scanf("%d", &array[c]);
    printf("Enter a number to search \n");
    scanf("%d", &search);
    
    for(c=0;c<n;c++)
    {
        if(array[c] == search) /* if requirede element is found */
        {
            printf("%d is present at location %d.\n",search, c+1);
            break;
        }
    }
    if (c==n)
      printf("%d isn't present in the array.\n",search);
      
      getch();
 }

File: Data Structure/Array Or Vector/Linear Search/SolutionByMashiyat.py
def linear_search(lst, key):
    for i in range(len(lst)):
        if lst[i] == key:
            return i
    return -1
 
lst = input('Enter the integers: ')
lst = lst.split()
lst = [int(x) for x in lst]
key = int(input('Enter a number to search: '))
 
index = linear_search(lst, key)
if index < 0:
    print('{} is not present in the array.'.format(key))
else:
    print('The number is present at location {}.'.format(key, index))
